assets/css/heygov-site.css

```diff
@@ -1,3 +1,18 @@
+h6 {
+	font-weight: 700;
+	font-size: 16px;
+	line-height: 100%;
+}
+h3 {
+
+	font-weight: 700;
+	font-size: 20px;
+	line-height: 36px;
+}
+
+h1, h2, h3, h4, h5, h6 {
```

Comment:
hey @SanjaBudic, all css in plugin .css files should be prefixed with `heygov-`, so these should be `.heygov-h1`. 

At the moment, these styles will override theme heading styles